frontend/pages/setting/password.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <div>
+    <v-form ref="passwordForm">
+      <UserFormPassword
+        v-model="user.password"
+        noIcon
+        noValidation
+        label="現在のパスワード"
+      />
+
+      <UserFormPassword
+        v-model="user.newpassword"
```

Comment:
特に理由がなければキャメルケースに合わせましょう。
```suggestion
        v-model="user.newPassword"
```

Reply:
修正しました。